MembershipListenerTest.initialMemberEvents_whenClusterRestarted

```
java.lang.AssertionError: null
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertNotNull(Assert.java:712)
	at org.junit.Assert.assertNotNull(Assert.java:722)
	at com.hazelcast.test.HazelcastTestSupport.assertInstanceOf(HazelcastTestSupport.java:782)
	at com.hazelcast.client.MembershipListenerTest.initialMemberEvents_whenClusterRestarted(MembershipListenerTest.java:278)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-ZuluJDK6-freestyle/154/testReport/junit/com.hazelcast.client/MembershipListenerTest/initialMemberEvents_whenClusterRestarted/

--- a/hazelcast-client/src/test/java/com/hazelcast/client/test/TestHazelcastFactory.java
+++ b/hazelcast-client/src/test/java/com/hazelcast/client/test/TestHazelcastFactory.java
@@ -96,7 +96,7 @@ public class TestHazelcastFactory extends TestHazelcastInstanceFactory {
                 Collection<InetSocketAddress> inetAddresses = new ArrayList<InetSocketAddress>();
                 for (Address address : getKnownAddresses()) {
                     Collection<InetSocketAddress> addresses = AddressHelper.getPossibleSocketAddresses(address.getPort(),
-                            address.getHost(), 3);
+                            address.getHost(), 1);
                     inetAddresses.addAll(addresses);
                 }
                 return inetAddresses;

